tests/Silex/Tests/Provider/FormServiceProviderTest.php

```diff
@@ -190,6 +198,14 @@ public function getExtendedType()
             return class_exists('Symfony\Component\Form\Extension\Core\Type\RangeType') ? 'Symfony\Component\Form\Extension\Core\Type\FileType' : 'file';
         }
 
+        /**
+         * @return string[]
+         */
+        public static function getExtendedTypes(): iterable
+        {
+            return [class_exists('Symfony\Component\Form\Extension\Core\Type\RangeType') ? 'Symfony\Component\Form\Extension\Core\Type\FileType' : 'file'];
```

Comment:
You could use :: fqcn and use statements
This way it would be clickable/followable if existing.